miniapps/nurbs/makefile

```diff
@@ -151,6 +180,6 @@ clean-build:
 	rm -rf *.dSYM *.TVD.*breakpoints
 
 clean-exec:
-	@rm -f refined.mesh sin-fit.mesh mesh.* sol.* mode_* naca-cmesh.mesh
-	@rm -rf Example1*
+	@rm -f refined.mesh sin-fit.mesh ex5.mesh mesh.* sol.* mode_* naca-cmesh.mesh sol_?.gf
```

Comment:
Does `sol_?.gf` really exist?

Reply:
These outputs are generated by nurbs_ex5 and nurbs_solenoidal.

In order for `make test` to clean-up after itself `sol_?.gf` is necessary. Moreover, I also added `exsol.mesh` to the clean statement.